### PR TITLE
CRO Unique Items Changes

### PR DESCRIPTION
**Pull request by AMcilraith**
## About The Pull Request
<details>
<summary>
Gives CRO Advanced AMI in exchange for minor unique equipment changes.
</summary>
<hr>

Provides the CRO with the Advanced AMI Control Module, in exchange for placing the unique Hydrogen-Plasma Sword inside the "Hydrogen-Plasma Fusion" node with the other hydrogen tech.
	
<hr>
</details>

## Changelog
:cl:
add: Adds Hydrogen-Plasma sword to hydrogen tech RnD node.
del: Removed Hydrogen-Plasma sword from CRO locker.
tweak: Replaced AMI control module with Advanced AMI control module in CRO locker.
/:cl:

### CODE REVIEW

File: code/modules/clothing/spacesuits/rig/suits/station.dm [hunk 1]
```diff
@@ -344,6 +344,15 @@ Technomancer RIG
 /obj/item/clothing/head/helmet/space/rig/advhazmat
 	camera_networks = list(NETWORK_RESEARCH)
 
+/obj/item/rig/advhazmat/equipped
+	req_access = list(access_rd)
+	drain = 10
```
Comment (Trilbyspaceclone) [inline code/modules/clothing/spacesuits/rig/suits/station.dm:349]:
```suggestion
```
No real reason to give it less drain

Comment (AMcilraith) [reply code/modules/clothing/spacesuits/rig/suits/station.dm:349]: Fair. I will fix.

**Comment by AMcilraith** [comment]
i forgot an "=" so the next push fixes that

**Comment by Trilbyspaceclone** [comment]
Your also going to have to update the sword dm to have correct code commits. 
Might also want to make sure it prints without a cell as well

**Comment by AMcilraith** [comment]
> Your also going to have to update the sword dm to have correct code commits. Might also want to make sure it prints without a cell as well

Elaborate? Sword.dm needs no changes due to the fab spawning it as it does with the locker (src) , does it not do that?

**Comment by AMcilraith** [comment]
Found the issue. Pushing fix

**Comment by Trilbyspaceclone** [comment]
commits in the dm of the sword say its cro uniquic and i want to spawn without a cell

**Comment by AMcilraith** [comment]
> commits in the dm of the sword say its cro uniquic and i want to spawn without a cell

Literally every hydrogen weapon in the game spawns with a cell. Why would this one be different?

**Comment by AMcilraith** [comment]
For consistency sake, it will spawn with a cell. 

No other hydrogen weapon does not spawn with a cell.

**Comment by AMcilraith** [comment]
This can be fixed in a separate PR.

**Comment by AMcilraith** [comment]
The other requested changes will be pushed when I'm back at my computer.

**Comment by Trilbyspaceclone** [comment]
compresses this in #5890

**Pull request closed without merging**